website_require_login/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Advitus MB
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Website Login Required",
+    "category": "Website",
+    "version": "14.0.1.0.0",
+    "author": "Advitus MB, Ooops, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/website",
+    "license": "LGPL-3",
+    "depends": [
+        "website_sale",
```

Comment:
Why is there an depends on sale? 
Maybe it's enough to add only a website?

Reply:
Fixed